android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -226,31 +264,14 @@ public void captureEnvelope(PluginCall call) {
             for (int i = 0; i < bytes.length; i++) {
                 bytes[i] = (byte) rawIntegers.getInt(i);
             }
-
-            final String outboxPath = HubAdapter.getInstance().getOptions().getOutboxPath();
-
-            if (outboxPath == null || outboxPath.isEmpty()) {
-                logger.info("Error when writing envelope, no outbox path is present.");
-                call.reject("Missing outboxPath");
-                return;
-            }
-
-            final File installation = new File(outboxPath, UUID.randomUUID().toString());
-
-            try (FileOutputStream out = new FileOutputStream(installation)) {
-                out.write(bytes);
-                logger.info("Successfully captured envelope.");
-            } catch (Exception e) {
-                logger.info("Error writing envelope.");
-                call.reject(String.valueOf(e));
-                return;
-            }
-        } catch (Exception e) {
-            logger.info("Error reading envelope.");
-            call.reject(String.valueOf(e));
-            return;
-        }
-        call.resolve();
+            InternalSentrySdk.captureEnvelope(bytes);
+            call.resolve();
+          }
+          catch (Throwable e) {
+            final String errorMessage ="Error while capturing envelope";
+            logger.log(Level.WARNING, errorMessage);
+            call.reject(errorMessage);
```

Comment:
I see previously this was `call.reject(String.valueOf(e));`? Does this change the error in any way?